c/zss.c

```diff
@@ -560,6 +567,41 @@ static void installWebPluginDefintionsService(WebPluginListElt *webPlugins, Http
   zowelog(NULL, LOG_COMP_ID_MVD_SERVER, ZOWE_LOG_DEBUG2, "end %s\n", __FUNCTION__);
 }
 
+static void parseLoggingData(HttpServer *server, ShortLivedHeap *slh, const char *filename) {
+
+  char errorBuffer[512] = { 0 };
+  JsonObject *jsonObject = NULL;
+  
+  Json *json = jsonParseFile(slh, filename, errorBuffer, sizeof(errorBuffer));
+  if (json) {
+    if (jsonIsObject(json)) {
+      jsonObject = jsonAsObject(json);
+    }
+    else {
+      zowelog(NULL, LOG_COMP_ID_MVD_SERVER, ZOWE_LOG_WARNING, "error while parsing %s: %s\n", errorBuffer);
```

Comment:
This will ABEND - the format string has two ```%s``` you pass one string. Same for the other similar calls.